server-adapters/resteasy-vertx/src/test/java/org/jboss/resteasy/test/JaxrsAsyncTest.java

```diff
@@ -1,191 +0,0 @@
-package org.jboss.resteasy.test;
-
-import static org.jboss.resteasy.test.TestPortProvider.generateURL;
-
-import io.netty.handler.codec.http.HttpHeaders.Values;
-
-import javax.ws.rs.client.Client;
-import javax.ws.rs.client.ClientBuilder;
-import javax.ws.rs.core.Response;
-
-import org.junit.Assert;
-import org.junit.BeforeClass;
-import org.junit.AfterClass;
-import org.junit.Before;
-import org.junit.After;
-import org.junit.Test;
-
-import javax.ws.rs.client.Invocation.Builder;
-
-/**
- * @author <a href="mailto:bill@burkecentral.com">Bill Burke</a>
- * @version $Revision: 1 $
- */
-public class JaxrsAsyncTest
-{
-   static String BASE_URI = generateURL("");
-   static Client client;
-
-   static final int REQUEST_TIMEOUT = 1000;
-
-   @BeforeClass
-   public static void setupSuite() throws Exception
-   {
-      VertxContainer.start().getRegistry().addSingletonResource(new AsyncJaxrsResource());

```

Comment:
Well, Netty 4 adaptor might have a similar test, but this is using VertxContainer, so it would make sense to keep it.